### ISSUE TITLE
[App]: Error with tables in offline mode

### ISSUE THREAD

**Issue by nicolas-dufour** [wandb/wandb#4611]
### Current Behavior

When logging images inside tables in offline mode, images get replace by the word Image

### Expected Behavior

The images should be displayed properly.
When logging in online mode, it works perfectly

### Steps To Reproduce

See the following table: 
https://wandb.ai/nicolas-dufour/Textual%20Style%20Content%20Extraction/reports/Weave-Content-prompts-22-12-12-18-50-59---VmlldzozMTMyMTU1?accessToken=jh93h5d88fanu68ovbnh290bmhyv5obp0my5kh4recl9hj6tuogrgek3yo076uwb

### Environment

OS: Linux

Browsers: Chrome

Version: 0.13.6

**Comment by lesliewandb** [comment]
Hi @nicolas-dufour, thank you for writing in! Can you tell me how you are logging your tables with images please? A code snippet will suffice.

**Comment by lucasdavid** [comment]
I'm also having this problem. It works fine in online mode, and images seem to be uploaded when I run `run.finish()` at the end of the experiment.
On offline mode, images are not uploaded when I do `wandb sync wandb/offline-...`, and they are nowhere to be found inside the `./wandb/offline-..../.../media` folder.

I'm logging it as this:
```py
# images = np.random.rand(4, 512, 512, 3)
wb_images = [wandb.Image(c) for c in images]
wb_cams = [wandb.Image(c) for c in cams]

columns = ("Image", "CAM", "Labels", "CG Predictions", "OC Predictions")
entries = (wb_images, wb_cams, wb_targets, wb_predics, wb_oc_pred)
columns_v, entries_v = [], []

for c, e in zip(columns, entries):
  if e is not None:
    columns_v += [c]
    entries_v += [e]

data = [list(row) for row in zip(*entries_v)]
table = wandb.Table(columns=columns_v, data=data)
wandb.log({"val/predictions": table}, commit=False)
```

**Comment by lesliewandb** [comment]
Thank you so much for this information! I looked into your project and I see that you're using an old version of wandb. Can you upgrade to our most recent version 0.13.6 and let me know if you're still running into this error?

**Comment by nicolas-dufour** [comment]
Hi @lesliewandb . I've had this error with the most recent version as well, before going back to 0.12.4 which has been the most stable version for me sor far. The error is however still present in 0.13.6.

**Comment by thanos-wandb** [comment]
Hi @nicolas-dufour @lucasdavid I wanted to follow up with you about this issue you've reported. I have let the engineering team know about this, and we will reach out to you here once there is a fix. 

In the meantime, there are two workarounds that could enable you to see the logged table with images. One is through the Artifact that is created when logging a `wandb.Table` object. For the example that @lucasdavid provided above, there will be an artifact created and the table will be visible in the artifact's Files tab and then inside `val/predictions.table.json` file. 

The second way would be to add a Weave panel in your run's workspace and execute an expression similar to: `project("ENTITY", "PROJECT").artifact("run-runid-valpredictions").membershipForAlias("v0").artifactVersion.file("val/predictions.table.json")`

I hope this helps. We will keep you posted on any updates.

**Comment by nicolas-dufour** [comment]
Hi thanos-wandb. Thanks for the temporary workarounds fixes, but none work for me, indeed, i want to be able to compare different tables and i couldn't found a way to compare artifacts. Maybe i'm missing something? 
Thank you for the help!

**Comment by thanos-wandb** [comment]
Hi @nicolas-dufour this issue hasn't been fixed yet from the engineering team. Regarding the provided workarounds, the way to compare artifact tables side-by-side is documented [here](https://docs.wandb.ai/guides/data-vis/tables#table-comparison). Would this work for you?

**Comment by nicolas-dufour** [comment]
Hi, the ressource linked here is already what i was doing when trying to visualize my tables and it's not displaying the images, like describide above.

**Comment by thanos-wandb** [comment]
Hi @nicolas-dufour sorry you're experiencing this issue could you provide me with a screenshot to understand more what might cause the issue? or your project name to check this for you? thanks

**Comment by nicolas-dufour** [comment]
Hi @thanos-wandb , here is what i'm seeing:
![image](https://user-images.githubusercontent.com/33259879/214015193-75a784e7-5729-4d0d-8d0c-7ca37ab22d57.png)

**Comment by thanos-wandb** [comment]
Hi @nicolas-dufour the original issue you've reported that renders "image" as text instead of the actual image hasn't been resolved yet. I will keep you updated here once the engineering team releases a fix for this. 

However, could you click on the left panel the Artifacts icon to take you to the artifact store, and then search for this logged table there? are you able to see the Images from the artifact's Files tab when clicking on the `table-name.json` file?

**Comment by nicolas-dufour** [comment]
Hi @thanos-wandb Yes i do see the table there, but the interface is much less intuitive to look between tables and finding the right run. I do hope that the bug gets solved in the main interface since it really limitates the use of Tables for academics that use offline clusters

**Comment by thanos-wandb** [comment]
Hi @nicolas-dufour thanks for the update, this was only meant as a temporary workaround in order to verify that your data are uploading fine. However, I agree this shouldn't happen and probably affects many users so I will try to bump up the priority on this one. I will keep you posted here!

**Comment by jaded0** [comment]
Yeah I've spent hours trying to figure out how to just show image results alongside inference steps. To me, that's really basic functionality.

Using the above workaround, I get the same image over and over again.
